packages/atomic/src/components/atomic-folded-result-list/atomic-folded-result-list.tsx

```diff
@@ -25,7 +25,7 @@ import {
 } from '../result-lists/result-list-common';
 
 /**
- * The `atomic-folded-result-list` component is responsible for displaying query results by applying one or more result templates.
+ * The `atomic-folded-result-list` component is responsible for displaying folded query results, by applying one or more result templates up to two levels nested (i.e. result, child and grandchild)
```

Comment:
```suggestion
 * The `atomic-folded-result-list` component is responsible for displaying folded query results, by applying one or more result templates for up to three layers (i.e., to the result, child and grandchild).
```
Assuming I correctly understood that the template applies not only to the child and grandchild, but also to the result. Otherwise I would recommend:
```suggestion
 * The `atomic-folded-result-list` component is responsible for displaying folded query results, by applying one or more result templates for up to two nested levels (i.e., to the children and grandchildren of results).
```

Reply:
I think the first one, as even though it doesn't use the same template for all levels, it still applies A template for each level